presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadata.java

```diff
@@ -275,8 +277,10 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
         String tableName = schemaTableName.getTableName();
 
         Schema schema = new Schema(toIceberg(tableMetadata.getColumns()));
+        AtomicInteger nextFieldId = new AtomicInteger(0);
```

Comment:
Is it ok to start fields at `0`, or do they need to start with `1`?

Reply:
I think it's ok because Iceberg will reassign field IDs anyway. But let's make it to `1` to be consistent with Iceberg's reassigning work.